### PR TITLE
Next auth git

### PR DESCRIPTION
**Pull request by thaistcosta**
Hello Chicas ❤️ 

I still need to make it more modular, but for now it's the basic login structure.

I didn't work on layout I just worked on functionality.

I changed the User schema and created some queries.

Let me know if you have any question or comments about it

### CODE REVIEW

File: server/prisma/schema.prisma [hunk 1]
```diff
@@ -7,22 +7,20 @@ generator client {
 
 datasource db {
   provider = "postgresql"
-  url      = env("DATABASE_URL")
-  shadowDatabaseUrl = env("SHADOW_DATABASE_URL")
+  url      = "postgres://efvubjdgccjxji:db21c7c801dde62acf6ea5de475602884e310f0101d5d8db07f5d0cee096124b@ec2-44-199-143-43.compute-1.amazonaws.com:5432/dfsrjkkvo1pbpk"
```
Comment (lexarisa) [inline server/prisma/schema.prisma:10]: shouldnt we put this in .env

**Review by lexarisa — APPROVED** [review]
<3 Great great work!

File: server/prisma/schema.prisma (hunk 1 above)
Comment (thaistcosta) [reply server/prisma/schema.prisma:10]: Omggggggg ! I forgooot

**Review by alexandra-plana — APPROVED** [review]
This is amazing!!! i am in awe S2

**Pull request merged**